### PR TITLE
Clean karim branch

### PR DESCRIPTION
**Pull request by KarimChukfeh**

### CODE REVIEW

File: sunspot-solr.pid [hunk 1]
```diff
@@ -0,0 +1 @@
+7740
```
Comment (tahnok) [inline sunspot-solr.pid]: I this supposed to be committed?

File: Gemfile [hunk 2]
```diff
@@ -50,7 +50,7 @@ group :test do
   gem 'database_cleaner', '~> 1.0.1'
   gem 'launchy', '~> 2.3.0'
   gem 'selenium-webdriver', '~> 2.39.0'
+  gem 'shoulda'
```
Comment (tahnok) [inline Gemfile]: Are you using this gem?

File: sunspot-solr.pid (hunk 1 above)
Comment (KarimChukfeh) [inline sunspot-solr.pid]: the file is generated and the pid changes every time you fire up solr.

File: Gemfile (hunk 2 above)
Comment (KarimChukfeh) [inline Gemfile]: unsure if i will be using it or not. it can be removed for now

**Pull request merged**